content/tutorial/tutorial.md

```diff
@@ -12,185 +12,190 @@ redirect_from:
 
-## Inspecting the Starter Code {#inspecting-the-starter-code}
+## स्टार्टर कोड का निरीक्षण {#inspecting-the-starter-code}
 
-If you're going to work on the tutorial **in your browser,** open this code in a new tab: **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. If you're going to work on the tutorial **locally,** instead open `src/index.js` in your project folder (you have already touched this file during the [setup](#setup-option-2-local-development-environment)).
+यदि आप **अपने ब्राउज़र में** ट्यूटोरियल पर काम करने जा रहे हैं, इस कोड को एक नए टैब में खोलें: **[स्टार्टर कोड।](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)**. यदि आप **लोकली** रूप से ट्यूटोरियल पर काम करने जा रहे हैं, तो अपने प्रोजेक्ट फ़ोल्डर में `src/index.js` खोलें ([सेटअप](#setup-option-2-local-development-environment) के दौरान आप इस फ़ाइल को पहले ही देख चुके हैं)।
 
-This Starter Code is the base of what we're building. We've provided the CSS styling so that you only need to focus on learning React and programming the tic-tac-toe game.
+यह स्टार्टर कोड हम जो निर्माण कर रहे हैं उसका आधार है। CSS स्टाइलिंग पपहले से ही प्रदान की हुई है ताकि आपको केवल React सीखने और 
+टिक-टैक-टो गेम की प्रोग्रामिंग पर ध्यान देने की आवश्यकता हो।
 
-By inspecting the code, you'll notice that we have three React components:
+कोड का निरीक्षण करने पर, आप देखेंगे कि हमारे पास तीन React कौम्पोनॅन्ट हैं:
 
 * Square
 * Board
 * Game
 
-The Square component renders a single `<button>` and the Board renders 9 squares. The Game component renders a board with placeholder values which we'll modify later. There are currently no interactive components.
+Square कौम्पोनॅन्ट एक अकेले `<button>` और Board के 9 वर्गों रेंडर करता है। Game कौम्पोनॅन्ट प्लेसहोल्डर वैल्यूज़ के साथ एक Board रेंडर करता है जिसे हम बाद में संशोधित करेंगे। वर्तमान में इस कोड में कोई इंटरैक्टिव कौम्पोनॅन्ट नहीं हैं।
 
-### Passing Data Through Props {#passing-data-through-props}
+### Props के माध्यम से डेटा पास करना {#passing-data-through-props}
 
-To get our feet wet, let's try passing some data from our Board component to our Square component.
+शुरुआत के लिए, आइए हम Board कौम्पोनॅन्ट से हमारे Square कौम्पोनॅन्ट को कुछ डेटा पास करने का प्रयास करते हैं।
 
-We strongly recommend typing code by hand as you're working through the tutorial and not using copy/paste. This will help you develop muscle memory and a stronger understanding.
+हम दृढ़ता से सलाह देते हैं की आप कोड हाथ से लिखें क्योंकि आप ट्यूटोरियल के माध्यम से काम कर रहे हैं और कॉपी / पेस्ट का उपयोग नहीं कर रहे हैं। यह आपकी मांसपेशियों की स्मृति और एक मजबूत समझ विकसित करने में मदद करेगा।
```

Comment:
की should be choti maatra

Reply:
काम कर रहे हैं -> seekh rahe hain

Reply:
और कॉपी / पेस्ट का उपयोग नहीं कर रहे हैं -> upyog na karna aapke liye faydemand rahega